tailwind.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    './ui/**/*.{js,jsx,ts,tsx}',
+    './node_modules/@metamask/design-system-react/**/*.{cjs,mjs}',
```

Comment:
We can probably remove this if we want as we are no longer importing the design system react library